firmware-management/updated/updated-client/source/rpc/Client.h

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) 2020 Arm Limited and Contributors. All rights reserved.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+#ifndef UPDATED_CLIENT_RPC_CLIENT_H
+#define UPDATED_CLIENT_RPC_CLIENT_H
+
+#include "updated-rpc/updated-rpc.grpc.pb.h"
```

Comment:
Is this file auto-generated by the `protoc`? If so, I think is worth adding a comment on all includes of auto-generated headers.